### ISSUE TITLE
Reducing The Amount Of Notifications that the Cops Get & To give criminals a hint/notification that the cops have been called

### ISSUE THREAD

**Issue by jay2huncho** [New-Chance-RP/Feedback#697]
I am requesting that you reduce the amount of notifications that the cops get, and to give the criminals a hint if the cops have been called.

I am requesting this, because the cops seem to get notified for every crime, even if the criminal is trying to be cautious. For example, I have a place out in the middle of nowhere where I like to make my drugs. There's no locals ever around, and somehow 3 cops responded to a call that I was cooking drugs... Also, for house robberies, there's no way to know if the cops are outside... We can't see out the windows, or hear any sirens. We can't even talk on the radio in the house robberies. Right when we walk out of the house, we have 4 cops just aiming guns at our heads. It's almost a 100% chance to get arrested after the house robbery when there's cops in, because all they do is power game the situation by camping outside, and they they search us, and find all the illegal shit on us. The only other way to get out of the situation, is to shoot them, and whenever you shoot them, they seem to get very salty... Another example is drugs. If I walk up to a local, and offer them drugs, they deny it. They don't pull out a phone and call the cops... The cops show up almost instantly sometimes. So like I said, give us a hint that they snitched. 

Positive Impacts:
-Gives both cops and criminals a fair chance
-Cops will actually have to patrol, and look our for suspicious activity, instead of always relying on the notification popping up on their screen

Negative Impacts:
-Can't think of any, but I'm sure some cops will reply with some

EXAMPLE OF WHAT I MEAN BY CRIMINAL GETS A HINT WHEN COPS HAVE BEEN CALLED: (I offer some drugs to a local, he rejects, and then when he walks away, he will run away, or maybe take a picture of your face, or even *Pulls out phone and calls cops*. Anything to give us a hint that the cops have been called.)

**Comment by wavyplays** [comment]
Ive heard a lot of locals say "Can I get an officer to my location" or something along those lines, are you requesting stuff like that to help you know the cops are en route or are you requesting us to get less notifications?

**Comment by GothiccBih** [comment]
I agree there needs to be something to balance the house robberies at least. For instance:

1.) Cops are only notified if you're being loud in the house. This includes running, having your talk volume to "shout or Talk" perhaps if possible. 

2.) Cops are only notified if a local is present. (If possible.) 

3.) Cops are notified that a home alarm has been triggered if a lock pick breaks or possibly if someone fails to successfully lockpick the home.

**Comment by jay2huncho** [comment]
@wavyplays I am requesting that we hear something like that every time the cops are called, and I am also requesting less notifications, because you guys get notified like every bag we sell lol.

**Comment by SwizzyWT** [comment]
I would have to agree with Jay on this one, house robberies 100% jail time no matter how you RP it coming out of this house, a noise meter needs to be applied to go off if we make to much noise. Drugs- not many people are going to call the cops on you in RL for selling some drugs in the hood and if they do in this situation a local gangster will let you know that someone has called the cops, cops need to be out patrolling areas where it looks like drugs will be sold at, tbh where I’m from if someone calls the cops for someone selling drugs they have just signed their death warrant.

**Comment by BadBritishGamer** [comment]
Appreciate someone finally making a genuine feedback about an issue, not just tagging on to other police based situations.

I have never done anything more than play cop in roleplay, I don't get to patrol as often anymore due to other duties that need to be fulfilled. 

When I created the PD I wanted it to be different, I wanted it to be a world where it's not Cops v Crims all the time. I wanted there to be good interactions that aren't all about arrests and shootouts.

I agree and it's something that as a command team we will work with the development team and work on a happy medium, Senlar is already looking to change the fact house robberies give 100% notifications as it's unrealistic. 

I think it also plays too far in the other direction suggesting that you should be in some way notified every time cops are called for any activity. 90% of people in life won't report you to the cops directly infront of you, they will move away out of view avoid confrontation and then call the cops and you'd be none the wiser.

But I agree if there is a way that for example 30% of the time cops don't get notified of "X" activity going down then we should get that going because it keeps the balance.

**Comment by GothiccBih** [comment]
> I agree there needs to be something to balance the house robberies at least. For instance:
> 
> 1.) Cops are only notified if you're being loud in the house. This includes running, having your talk volume to "shout or Talk" perhaps if possible.
> 
> 2.) Cops are only notified if a local is present. (If possible.)
> 
> 3.) Cops are notified that a home alarm has been triggered if a lock pick breaks or possibly if someone fails to successfully lockpick the home.

I forgot to add, if there are gonna be house alarms, maybe a tier of difficulty? Maybe breaking into a home in Paleto some of the houses have alarms, some don't. But in areas like Vinewood Hills most of them have alarms, but higher chances of better loot maybe?

**Comment by Senlarus** [comment]
It'll happen and is something I've already planned on doing.

**Comment by Senlarus** [comment]
I designed the cop notifications to not make them OP. I really don't want to go into detail because of meta and also wanting to protect some of the internal system designs that make our server a little more unique... but I will say this.

- Notifications to police only report an area (Big red circle shows up, not a pinpoint map marker)
- 90% of the crimes reported allow for a window of time for the crim to stop the ped from calling the cops.
- 17% is the rate at which drug sales send notifications to police. Very low
- Some of the "NEW" criminal systems still need to be built into my custom notification system as they are not yet there. So please give me time to complete this work and things will get better.

**Comment by Zopfco** [comment]
> * 17% is the rate at which drug sales send notifications to police. Very low

Is it possible to make this more varied, so that there's a difference between (eg) selling in an alley at 2am vs in front of a church at noon?

**Comment by ChuckBurn** [comment]
17% seems very balance. Out of 100 bags / 20k  cops will get 17 alerts.

**Comment by Senlarus** [comment]
I can make it more varied but it's not worth the programming effort. The 17% AOE alert (not pinpoint) is really just to let cops know there's drug activity in an area so they can respond and do some monitoring to try to find it. It's not meant to bring them right to a criminal and can easily create some nice hair raising interactions where a crim ducks into an ally or bush right before a cop rolls by and is none the wiser.
